.env.example

```diff
@@ -19,13 +20,54 @@ CHRONOS_SCHEDULE_JOBS='0 */10 * * * *'
 CHRONOS_UPDATE_TOKENS='0 0 0 1,15 * *'
 CHRONOS_PROCESS_RECURRENCE='0 */5 * * * *'
 CHRONOS_AUTOPAUSE='0 0 5 * * *'
-CI='true'
+# Enter a short url redirect service for invitations, it needs to redirecto to /invitation-link
+# INVITATION_SHORTLINK='example.com'
+
+# AUTHENTICATION
+AUTH_INTERNAL_DISABLED='false'
+AUTH_GOOGLE_DISABLED='false'
+AUTH_SSO_DISABLED='false'
+OAUTH2_REDIRECT=''
+
+# MONITORING
 DD_AGENT_HOST='172.17.0.1'
 DD_API_KEY='DD_API_KEY'
 DD_APM_ENABLED='false'
 DD_TRACE_ENABLED='false'
 DD_ENV='local'
+# SEGMENT_WRITE_KEY='key_SEGMENT_WRITE_KEY'
+SENTRY_AUTH_TOKEN='key_SENTRY_AUTH_TOKEN'
+# SENTRY_DSN='key_SENTRY_DSN'
+
+# CDN SETTINGS
```

Comment:
I think this is only partially correct. If I interpret [this](https://github.com/ParabolInc/parabol/blob/master/packages/server/fileStorage/README.md#s3-configuration) correctly, we're using the other AWS settings for `yarn predeploy` for uploading the artefacts. So it's CDN related, but not for the runtime.

Reply:
You are correct TODAY. If in the future we use any other AWS feature, as we do for Google, we would use the same variables. That is why I suggested putting if on that way.

That said, right now it would be ok too having them in the CDN part.

Reply:
I finally went with separating AWS as we separate GOOGLE variables, and keeping the AWS_S3_BUCKET and the GOOGLE_GCS_BUCKET under the CDN section.